src/views/InputView.js

```diff
@@ -0,0 +1,9 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+
+class InputView {
+  static async readLineAsync(message) {
+    return MissionUtils.Console.readLineAsync(message);
+  }
```

Comment:
`MissionUtils.Console.readLineAsync(message);` 코드가 promise를 리턴하는 함수라서 readLineAsync 함수에 async를 붙이지 않아도 원하는 대로 작동할 거 같습니다!

```suggestion
  static readLineAsync(message) {
    return MissionUtils.Console.readLineAsync(message);
  }
```

Reply:
그러네요! async를 붙이지 않아도 됐는데 비동기만 생각하다 이부분 체크하지 못한 것 같습니다. 짚어주셔서 감사합니다.